inc/builder/controllers/class-astra-builder-ui-controller.php

```diff
@@ -276,21 +276,43 @@ public static function render_site_identity( $device ) {
 		 */
 		public static function render_mode_switcher() {
 
-			$switcher_icon  = astra_get_option( 'mode-switcher-icon-type' );
-			$switcher_label = astra_get_option( 'mode-switcher-label' );
-
+			$switcher_style       = astra_get_option( 'dark-mode-switch-style', 'icon' );
+			$switcher_light_icon  = astra_get_option( 'mode-switcher-light-icon' );
+			$switcher_dark_icon   = defined( 'ASTRA_EXT_VER' ) ? astra_get_option( 'mode-switcher-dark-icon' ) : $switcher_light_icon;
+			$switcher_dark_icon   = $switcher_light_icon; // Temp code for debugging, remove once 'mode-switcher-dark-icon' config added.
```

Comment:
```suggestion
			$switcher_dark_icon   = $switcher_light_icon; // Temp code for debugging, remove once 'mode-switcher-dark-icon' config added.
```
@imnavanath Please check if this needs to be removed.

Reply:
Removed code already in one of sub-pr, also verified this is not showing this in current PR 👍